contracts/seed/Seed.sol

```diff
@@ -313,16 +321,16 @@ contract Seed {
     function close() public onlyAdmin isActive {
         // transfer seed tokens back to admin
         if(minimumReached){
-            // amount of seed tokens to be distributed = seedAmountAtStart - seedRemainder
-            uint256 seedToTransfer = seedAmountAtStart.sub(seedAmountAtStart.sub(seedRemainder));
+            // remaining seeds - seedRemainder + feeSeedRemainder
```

Comment:
should it be `=` here?

Reply:
Okay yes, I used it as hyphen. Will improve it.